src/pages/gateway/getting-started.md

```diff
@@ -28,14 +28,14 @@ You will also need to have:
 -  An API key (provided by Adobe)
 -  An API to integrate (for example, your own API, any public OpenAPI REST endpoint, or an [Adobe Experience Manager API])
 
-## Configure your environment
+## Configure your environment (optional)
```

Comment:
```suggestion
## Configure your environment
```

Reply:
Step is needed for users to install the CLI and interact with their Meshes.

Steps 2 and 3 are optional. CLI will work out of the box after install (in step 1)